features/car/conditional_restrictions.feature

```diff
@@ -0,0 +1,447 @@
+            | jp    | yes    |
+
+        And the relations
+            | type        | way:from | way:to | node:via | restriction:conditional                |
+            | restriction | ej       | nj     | j        | only_right_turn @ (has_pygmies > 10 p) |
+
+        When I route I should get
+            | from | to | route    |
+            | e    | s  | ej,js,js |
+            | e    | n  | ej,nj,nj |
+            | e    | p  | ej,jp,jp |
+
+    @no_turning @conditionals
+    Scenario: Car - Restriction would be on, but the restriction was badly tagged
+        Given the extract extra arguments "--parse-conditional-restrictions=1"
+                                            # time stamp for 10am on Tues, 02 May 2017 GMT
+        Given the contract extra arguments "--time-zone-file=test/data/tz_world.shp --parse-conditionals-from-now=1493719200"
+        Given the customize extra arguments "--time-zone-file=test/data/tz_world.shp --parse-conditionals-from-now=1493719200"
+
+        Given the node map
+            """
+              n
+           p
+              j
+                 m
+              s
+            """
+
+        And the ways
+            | nodes |
+            | nj    |
+            | js    |
+            | pjm   |
+
+        And the relations
+            | type        | way:from | way:to | node:via | restriction:conditional             |
+            | restriction | nj       | pjm    | j        | no_left_turn @ (Mo-Fr 07:00-10:30)  |
+            | restriction | js       | pjm    | j        | no_right_turn @ (Mo-Fr 07:00-10:30) |
+
+        When I route I should get
+            | from | to | route      |
+            | n    | m  | nj,pjm,pjm |
+            | s    | m  | js,pjm,pjm |
+
+    @no_turning @conditionals
+    Scenario: Car - ignores except restriction
+        Given the extract extra arguments "--parse-conditional-restrictions=1"
+                                            # time stamp for 10am on Tues, 02 May 2017 GMT
+        Given the contract extra arguments "--time-zone-file=test/data/tz_world.shp --parse-conditionals-from-now=1493719200"
+        Given the customize extra arguments "--time-zone-file=test/data/tz_world.shp --parse-conditionals-from-now=1493719200"
+        Given the node map
+            """
+              n
+            p j e
+              s
+            """
+
+        And the ways
+            | nodes | oneway |
+            | nj    | no     |
+            | js    | no     |
+            | ej    | no     |
+            | jp    | no     |
+
+        And the relations
+            | type        | way:from | way:to | node:via | restriction:conditional               | except   |
+            | restriction | ej       | nj     | j        | only_right_turn @ (Mo-Su 08:00-12:00) | motorcar |
+            | restriction | jp       | nj     | j        | only_left_turn @ (Mo-Su 08:00-12:00)  | bus      |
+
+        When I route I should get
+            | from | to | route          |
```

Comment:
By adding a column with a `#` header, we could describe what each row is supposed to test.